### PR TITLE
Resume Registry Raw Format .txt rendering fix Ref: Issue #81

### PR DESCRIPTION
**Pull request by tylermneher**
Should fix the issue I raised [The Registry's .txt Raw Format Incorrectly Renders Location, Phone, and Email Information #81](https://github.com/jsonresume/jsonresume.org/issues/81).

### CODE REVIEW

**Comment by levino** [comment]
Hey @tylermneher, thank you for your contribution and for finding the bug. Unfortunately we will have to wait for the clean up PR #83 to be merged first and then we can proceed here. Please be patient for a bit.

File: apps/registry/pages/api/formatters/text.js [hunk 1]
```diff
@@ -3,9 +3,18 @@
 const format = async function format(resume) {
   let content = `
 ${resume.basics.name}
-Pittsburgh, PA 15201
-(555) 555-5555
-example@example.com
+${resume.basics.label}
+
+// Pittsburgh, PA 15201
```
Comment (levino) [inline apps/registry/pages/api/formatters/text.js:8]:
```suggestion
```

File: apps/registry/pages/api/formatters/text.js [hunk 2]
```diff
@@ -3,9 +3,18 @@
...
+${resume.basics.location.address}
+${resume.basics.location.city}
+${resume.basics.location.region}
+${resume.basics.location.postalCode}
+${resume.basics.location.countryCode}
+// (555) 555-5555
```
Comment (levino) [inline apps/registry/pages/api/formatters/text.js:14]:
```suggestion
```

File: apps/registry/pages/api/formatters/text.js [hunk 3]
```diff
@@ -3,9 +3,18 @@
...
+${resume.basics.phone}
+// example@example.com
```
Comment (levino) [inline apps/registry/pages/api/formatters/text.js:16]:
```suggestion
```

**Review by levino — CHANGES_REQUESTED** [review]
Please remove all dead code.

**Comment by thomasdavis** [comment]
@tylermneher not overly bothered with the comments being in there but you might as well remove them and then we can merge.

great work nontheless

**Review by levino — APPROVED** [review]

**Pull request merged**